queries.sql

```diff
@@ -14,4 +14,50 @@ SELECT * FROM animals WHERE neutered=true;
 
 SELECT * FROM animals WHERE NOT name='Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT * FROM animals;
+
+BEGIN;
+
+ALTER TABLE animals ADD species VARCHAR(50);
```

Comment:
Please move this command to modify `animals` table from here to `schema.sql` file as everything that creates or change the tables structure should be saved in the `schema.sql` file.

Reply:
Thanks 😊
Modified